tests/testDefect.py

```diff
@@ -31,10 +32,8 @@
 import lsst.afw.display.utils as displayUtils
 import lsst.ip.isr as ipIsr
 
-try:
-    type(display)
-except NameError:
-    display = False
+# set to True to display images in ds9
+display = False
```

Comment:
Not sure if I like either solution. I assume the former was to allow this to be imported and pick up `display` if it existed?